app/views/homepage/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<style type="text/css">
+ table {
```

Comment:
It would also be good to extract CSS into separate files rather than add them to views